marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/CallBatcherImpl.java

```diff
@@ -553,6 +570,11 @@ private void retry(CallManager.CallEvent event, boolean callFailListeners) {
         public CallBatcherImpl<CallManager.CallArgs, E> forArgs() {
             return new CallBatcherImpl(client, caller, CallManager.CallArgs.class).finishConstruction();
         }
+
+        @Override
+        public CallBatcher<CallArgs, E> forArgsGenerator(CallArgsGenerator<E> generator) {
+            return new CallBatcherImpl(client, caller, java.lang.Void.class, generator);
```

Comment:
Given that Void is imported in the file, is there any reason to use the package name prefix?